Dockerfile

```diff
@@ -2,9 +2,9 @@ FROM mcr.microsoft.com/playwright:v1.32.2-jammy as dev
 RUN mkdir /src
 WORKDIR /src
 COPY app/package.json app/yarn.lock ./
+RUN apt-get update && apt-get install -y --no-install-recommends openjdk-11-jre-headless curl build-essential libcairo2-dev libpango1.0-dev libjpeg-dev libgif-dev librsvg2-dev
```

Comment:
We could probably ditch libgif and libjpeg - what we use canvas for is rendering the puzzle preview images that populate the og:image metatag and we only use svg/png for that. But I doubt that will have any significant savings in image build time or anything so probably fine to keep.